framework/src/modules/random/module.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { LiskValidationError, validator } from '@liskhq/lisk-validator';
+import { BlockGenerateContext } from '../../node/generator';
+import {
+	BlockAfterExecuteContext,
+	BlockVerifyContext,
+	GenesisBlockExecuteContext,
+} from '../../node/state_machine';
+import { BaseModule, ModuleInitArgs } from '../base_module';
+import { RandomAPI } from './api';
+import { MAX_LENGTH_REVEALS_MAINCHAIN, MODULE_ID_RANDOM } from './constants';
+import { RandomEndpoint } from './endpoint';
+import { randomModuleConfig } from './schemas';
+import { ModuleConfig } from './types';
+
+export class RandomModule extends BaseModule {
+	public id = MODULE_ID_RANDOM;
+	public name = 'random';
+	public api = new RandomAPI(this.id);
+	public endpoint = new RandomEndpoint(this.id);
+
+	private _maxLengthRevealMainchain!: number;
```

Comment:
```suggestion
	private _maxLengthReveal!: number;
```
as it is not only for mainchain